learning_assistant/views.py

```diff
@@ -236,6 +236,6 @@ def get(self, request, course_run_id):
         user = request.user
 
         message_count = int(request.GET.get('message_count', 50))
-        message_history = get_message_history(courserun_key, user, message_count)
+        message_history = reversed(list(get_message_history(courserun_key, user, message_count)))  # Reversing order
```

Comment:
instead of reversing it here, can we just change the ordering here? https://github.com/edx/learning-assistant/blob/9153aa142435469f0bb14ae75c1f18ab39438f1e/learning_assistant/api.py#L221

Reply:
We can. It was one or the other. I'll make the change.

Reply:
Moved the reverse logic to the API call and updated corresponding tests.